### DISCUSSION TITLE
CS50W: Submitting Project O: Permission to me50/USERNAME.git denied to USERNAME

### DISCUSSION THREAD

**Discussion by SplendidDigital** [community/community#130295] [Programming Help]
### Select Topic Area

Question

### Body

Source: https://cs50.harvard.edu/web/2020/projects/0/

Unlike CS50x, it appears CS50W does not come with a Github/Codespace administered/owned by CS50.

So after creating a repository and adding files, I followed these steps in order to submit (push method):

git init

git add .

git commit -m "Initial commit"

git remote add origin https://github.com/me50/USERNAME.git (repaced by my Github username)

git checkout -b web50/projects/2020/x/search

Facing hurdle in this step:

6. git push -u origin web50/projects/2020/x/search

https://preview.redd.it/cs50w-submitting-project-o-permission-to-me50-denied-to-v0-v07be5up5b9d1.png?width=1024&format=png&auto=webp&s=29f03205b11cf207057313a50ac9917c9cb899e1

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @SplendidDigital ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
